DevCGlobalDirectory.md

```diff
@@ -372,6 +372,18 @@
   - Github: https://github.com/Devc-Douala
 </details>
 
+<details>
+  <summary>
+    <b>DevC Yaounde</b>
+  </summary>
+
+  - Lead(s):
+    - [Aurelien Xavier](https://github.com/xavier-pay)
+    - [Suzy Tresor Tchegnelene](#)
```

Comment:
Thanks for this amazing PR!

Can you please add the GitHub profile link of this lead?

Reply:
Hi, She recently got a Github... she actively uses **bitbucket**. Adding her Github profile link!